docs/faq.rst

```diff
@@ -37,6 +37,11 @@ which details your dependencies. See our guide on :ref:`guides/reproducible-buil
 You can also set your project documentation to install your project itself
 as a dependency.
 
+Additionally, your build may depend on extensions that require external,
+non-Python libraries to be installed. If you are using a :ref:`configuration
```

Comment:
Dependencies can be executables and various packages as well. Perhaps this would be the most clear describing libraries as system packages or something similar instead?

```suggestion
Additionally, your build may depend on extensions that require additional
system packages to be installed. If you are using a :ref:`configuration
```

Does that sound better?